src/test/java/seedu/address/logic/parser/SortStudentCommandParserTest.java

```diff
@@ -27,30 +28,37 @@ public void parse_emptyArg_throwsParseException() {
     @Test
     public void parse_validArgs_returnsSortStudentCommand() {
         // valid name
-        SortStudentCommand command = new SortStudentCommand("name");
+        SortStudentCommand command = new SortStudentCommand(SortStudentParameter.NAME);
         assertParseSuccess(parser, " " + PREFIX_SORT_BY + "name", command);
 
         // valid name
-        command = new SortStudentCommand("id");
+        command = new SortStudentCommand(SortStudentParameter.STUDENTID);
         assertParseSuccess(parser, " " + PREFIX_SORT_BY + "id", command);
 
         // valid name
-        command = new SortStudentCommand("email");
+        command = new SortStudentCommand(SortStudentParameter.EMAIL);
         assertParseSuccess(parser, " " + PREFIX_SORT_BY + "email", command);
```

Comment:
Is the comments updated to reflect the parameter being passed in?

Reply:
Good catch!